src/jsx/components/results_page.jsx

```diff
@@ -77,11 +92,10 @@ class ResultsPage extends React.Component {
 
   render() {
     var self = this;
-
     if (!this.state.json) return <div />;
     return (
       <div>
-        {this.props.hyphy_vision ? (
+        {this.props.platform == "hyphyVision" ? (

```

Comment:
I'm not sure that the NavBar should be a part of the HyPhyVision library at all.

Reply:
I agree. I moved the NavBar into `app.jsx` which is not part of the library.